### DISCUSSION TITLE
How to put the mouse in a specific position x,y and click on it?

### DISCUSSION THREAD

**Discussion by diegoparra** [sezanzeb/input-remapper#249] [Ideas]
Hey Guys, I'm trying to set up this condition, when I press the wheel down it should move to a specific part of my screen and then auto left click whenever my current position is. Any ideas on how to do it?

**Comment by sezanzeb** [comment]
Since the mouse position is not known to input-remapper, all you can do is first move the mouse to a corner, and then to the coordinates you want it to move to.

```
e(EV_REL, REL_X, -9000).
e(EV_REL, REL_Y, -9000).
e(EV_REL, REL_X, 100).
e(EV_REL, REL_Y, 100).
key(BTN_LEFT)
```

this moves the mouse to 100, 100 (measured from the top left) and clicks

**Comment by amarcinkowski** [comment]
Is it possible to move back the mouse to where it was? 
 To get somehow current position before moving it to 100,100 and click, then back to where it was as in example above?

**Reply by sezanzeb** [reply]
Not that I know. Input-remapper certainly can't do it

**Comment by sabudum** [comment]
You can use some third-party program to read the mouse coords, create a script with your entire algorithm, then launch the script from a key-combo, remap that key-combo to your preferred input device.

In fact, you wouldn't even need input-remapper for this
